packages/gatsby-theme-newrelic/src/components/Feedback.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import Button from './Button';
+import Icon from './Icon';
+
+const ALIGNMENT = {
+  LEFT: 'left',
+  CENTER: 'center',
+};
+
+const iconStyles = css`
+  margin-right: 0.5rem;
+`;
+
+const Feedback = ({ align, onPositiveClick, onNegativeClick, message }) => {
+  const [submitted, updateSubmitted] = useState(false);
+
+  return (
+    <div
+      css={css`
+        text-align: ${align};
+      `}
+    >
+      <p>{message}</p>
+      <div
+        css={css`
+          display: flex;
+          justify-content: ${align};
+
+          button {
+            margin-right: 0.5rem;
+          }
+        `}
+      >
+        <Button
+          onClick={() => {
+            updateSubmitted(true);
+            onPositiveClick();
+          }}
+          variant={Button.VARIANT.PRIMARY}
+          disabled={submitted}
```

Comment:
Do we want users to be able to submit multiple bits of feedback? It seems a bit odd that we'd only want them to submit 1 bit of feedback, then prevent them from doing so again.

Perhaps we could use the `useTimeout` hook in combination with this state to re-enable it after a set timeout?

Reply:
I wanted to gate the user feedback so that someone didn't spam on the buttons and thought that, when a user navigated to a new page the component would reset itself (was building this with the rail in mind).

Reply:
That makes sense. I think we should consider disabling it for like a second or 2, then re-enabling. I've been watching feedback from the launch, and there are times where the same person will add multiple feedback items for the same page as they visit. I personally would rather not prevent someone from giving us feedback just because they've already provided it for the current page. I do agree though that we should try and avoid the user spamming the buttons by clicking them a bunch of times in a row.

Reply:
As currently implemented, is it once per page, or once for the entire site?

I think it makes sense to make it once per page so people can't spam, but perhaps we could show the buttons again when the page changes since maybe the content improved (or got worse).

Reply:
@djsauble that was my intention behind the current functionality: navigating to a new page would "enable" them again.

That said, it might not be ideal for feedback on search / anything more dynamic.

Reply:
I was thinking we wouldn't show these feedback buttons on search. We can infer that search wasn't helpful if you don't click on anything.